vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/ProjectConfig.cs

```diff
@@ -1554,7 +1564,27 @@ internal bool GetUTDCheckOutputs(ref HashSet<string> outputs, HashSet<string> in
                 outputs.Add(Utilities.CanonicalizeFileNameNoThrow(pdbPath));
             }
 
-            return true;
+            var appConfig = inputs.FirstOrDefault(x => String.Compare(Path.GetFileName(x), "app.config", StringComparison.OrdinalIgnoreCase) == 0);
+            if (appConfig != null)
+            {
+                inputs.Remove(appConfig);
+                var exeConfig = Utilities.CanonicalizeFileNameNoThrow(outputAssembly + ".config");
+                directMappings.Add(Tuple.Create(appConfig, "PreserveNewest", exeConfig));
+            }
```

Comment:
i handle app.config (and content items) as 1 input -> 1 output. See my comment below